docs/src/pages/guides/notifications.md

```diff
@@ -80,19 +80,14 @@ Type of Notification in this case it would be set as `mailgun`.
 The detail settings needed to send a mailgun.
 
 **Recipients**
-Email addresses that will accept the mail sent from your mailgun account, the recipients address is set in array making it able to send to multiple addresses. `["monika@testmail.com", "symon@testmail.com"]`
+An array of email addresses that will receive the e-mail from Monika. ex. `["monika@testmail.com", "symon@testmail.com"]`
 
 **Api Key**
 Mailgun's account api key, mailgun's registered key to identify your account. ex. `70e34aba-0ea908325`
 
-**Domain**
-A domain that you have set in mailgun. ex. `sandboxmail.mailgun.com`
```

Comment:
for consideration: I think it will be better if using table

Reply:
updated, please check